src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -157,9 +176,11 @@ class MediaBaseViewer extends BaseViewer {
     load() {
         super.load();
 
+        this.addEventListenersForMediaLoad();
+
         const template = this.options.representation.content.url_template;
         this.mediaUrl = this.createContentUrlWithAuthParams(template);
-        this.mediaEl.addEventListener('loadeddata', this.loadeddataHandler);
+
         this.mediaEl.addEventListener('error', this.errorHandler);
```

Comment:
Should this live in `addEventListenersForMediaLoad`?

Reply:
This one can't because it isn't used in the `DashViewer`